src/ripple/app/ledger/InboundLedgers.cpp

```diff
@@ -250,6 +257,23 @@ class InboundLedgersImp
         mLedgers.clear();
     }
 
+    std::size_t fetchRate()
+    {
+        std::lock_guard<std::mutex
+            > lock(fetchRateMutex_);
```

Comment:
I love Fibonacci numbers and Phi is wonderful but this reads like the mutex is greater than a lock.